lms/envs/common.py

```diff
@@ -2035,6 +2035,7 @@ def _make_locale_paths(settings):
 
     # Database-backed configuration
     'config_models',
```

Comment:
is this an app that we're importing from django?

Reply:
It's `django-config-models` which we wrote (and is published to pypi).